TOOLING.md

```diff
@@ -4,13 +4,46 @@ Here you will find a list of tools used by the Pulsar team and information about
 
 ## Continuous Integration
 
-[Cirrus CI](https://cirrus-ci.com/github/pulsar-edit/pulsar) is used for Pulsar's continuous integration as well as for building application binaries.
+### [Cirrus CI](https://cirrus-ci.com/github/pulsar-edit/pulsar)
+
+Cirrus CI is used for Pulsar's continuous integration as well as for building application binaries.
+
+#### [Codacy](https://app.codacy.com/gh/pulsar-edit/repositories)
```

Comment:
```suggestion
### [Codacy](https://app.codacy.com/gh/pulsar-edit/repositories)
```
Move it from H4 to the same H3 level as Cirrus as it isn't a subsection of it?

Reply:
Good catch, it is not intended to be a subsection.